client/src/containers/backend/Form/UserClaims.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component, PropTypes } from 'react';
+import { Form, Text } from 'components/backend';
+import { makersAPI, usersAPI } from 'api';
+import { connect } from 'react-redux';
+import { entityStoreActions } from 'actions';
+import { entityUtils } from 'utils';
+import get from 'lodash/get';
+import capitalize from 'lodash/capitalize';
+
+const { request, flush, requests } = entityStoreActions;
+
+class FormUserClaims extends Component {
```

Comment:
Actually, as I look through this, I'm confused — it seems like this container is only used on Users/Edit.js. If it's not also useful on the other side of the equation, in Makers/Edit.js, then why are we abstracting this into its own container? Maybe the maker side isn't finished?